local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -94,6 +98,16 @@ func (u *volumeUtil) DeleteContents(fullPath string) error {
 	return nil
 }
 
+// GetFsAvailable returns available capacity about a mounted filesystem. fullPath
+// is the pathname of any file within the mounted filesystem.
+func (u *volumeUtil) GetFsAvailable(fullPath string) (uint64, error) {
+	var s syscall.Statfs_t
+	if err := syscall.Statfs(fullPath, &s); err != nil {
+		return 0, err
+	}
+	return uint64(s.Frsize) * s.Bavail, nil
```

Comment:
Hm is there a field that can distinguish between system reserved, and user used?